### PR TITLE
Challenges 0, 1, and 2 in Python

### PR DESCRIPTION
**Pull request by OkraArmband**
I did the thing! Comments/criticism welcome!

### CODE REVIEW

File: challenge_2/python/OkraArmband/src/unique_ele.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from collections import defaultdict
+
+def unique(lst):
+    # Super duper lazy way of solving problem... o(n^2) method.
+    unique_lst = [itm for itm in lst if lst.count(itm) == 1]
+    return unique_lst
+
+def unique_v2(lst):
```
Comment (jarydremillard) [inline challenge_2/python/OkraArmband/src/unique_ele.py]: Try to refrain from using ambiguous function names. Instead, make the name of the function accurate and descriptive of what it is about. For example, `def findUniqueChar():`

File: challenge_2/python/OkraArmband/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The program "unique_ele.py" has two methods in it: unique() and unique_v2(). Both of these functions carry out the same purpose and will function identically.
```
Comment (jarydremillard) [inline challenge_2/python/OkraArmband/README.md]: Please try to add how to run this file, as well as test inputs and outputs for someone can fully understand what this program is doing. Good use of everything else though!

**Comment by jarydremillard** [comment]
- [ ] Code review
- [ ] Code documented
- [ ] Correct path
- [ ] Code tested
- [ ] Check for conflicts
- [ ] **Merge PR**

File: challenge_2/python/OkraArmband/src/unique_ele.py (hunk 1 above)
Comment (OkraArmband) [inline challenge_2/python/OkraArmband/src/unique_ele.py]: Yep, that makes sense. Thanks very much!

**Pull request merged**